### ISSUE TITLE
[BUG] No readings on Sportstech SX600

### ISSUE THREAD

**Issue by Chriswhite69** [cagnulein/qdomyos-zwift#1872]
Turn the bike on and open the app, app finds bike okay, press start and pedal no reading shown on the screen, all stays at zero, bike starting the session okay with a resistance of 10, 

Working on a Apple iPad Pro 10.5 version 17.1.2
[debug-Sun_Dec_10_10_00_47_2023.log](https://github.com/cagnulein/qdomyos-zwift/files/13627342/debug-Sun_Dec_10_10_00_47_2023.log)

**Comment by cagnulein** [comment]
could you please also enable the fit plus bike setting and restart qz?

**Comment by Chriswhite69** [comment]
Still didn’t work
[debug-Sun_Dec_10_10_42_45_2023.log](https://github.com/cagnulein/qdomyos-zwift/files/13627426/debug-Sun_Dec_10_10_42_45_2023.log)

**Comment by cagnulein** [comment]
sorry the log is truncated, could you please upload a new one?
I will be travel the whole day , I will answer you back later today
Thanks

Il giorno dom 10 dic 2023 alle 11:44 Chriswhite69 ***@***.***>
ha scritto:

**Comment by Chriswhite69** [comment]
just tried it with fit plus and sx600 bike switched on and toorx/iconsol bike switched of and it started working. 👍

[debug-Sun_Dec_10_11_08_23_2023.log](https://github.com/cagnulein/qdomyos-zwift/files/13627479/debug-Sun_Dec_10_11_08_23_2023.log)

**Comment by cagnulein** [comment]
Glad to hear it!

Il giorno dom 10 dic 2023 alle 12:16 Chriswhite69 ***@***.***>
ha scritto:

**Comment by simonhaste-svg** [comment]
Still not showing any data changes

**Comment by cagnulein** [comment]
Share a debug log
https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

Roberto Viola

I sometimes trade time with my daughter to help. If this was valuable,
please support my time:
https://buymeacoffee.com/cagnulein
https://www.patreon.com/cagnulein/join
https://paypal.me/cagnulein

Software engineer and open source enthusiast, QZ Creator
http://robertoviola.cloud
http://qzfitness.com/


Il giorno dom 8 feb 2026 alle 13:49 simonhaste-svg ***@***.***>
ha scritto:

**Comment by simonhaste-svg** [comment]
[debug-Sun_Feb_8_16_55_37_2026.log](https://github.com/user-attachments/files/25163123/debug-Sun_Feb_8_16_55_37_2026.log)

**Comment by cagnulein** [comment]
Can you please enable the fit plus bike setting in the bike options and try
again?

Roberto Viola

I sometimes trade time with my daughter to help. If this was valuable,
please support my time:
https://buymeacoffee.com/cagnulein
https://www.patreon.com/cagnulein/join
https://paypal.me/cagnulein

Software engineer and open source enthusiast, QZ Creator
http://robertoviola.cloud
http://qzfitness.com/


Il giorno dom 8 feb 2026 alle 18:20 simonhaste-svg ***@***.***>
ha scritto:

**Comment by simonhaste-svg** [comment]
Data now showing thanks but Howdon i use mywhoosh with it please

**Comment by cagnulein** [comment]
Mywhoosh rowing compatibility is work in progress. I will finish next week.
Follow this
https://github.com/cagnulein/qdomyos-zwift/pull/4266


Roberto Viola

I sometimes trade time with my daughter to help. If this was valuable,
please support my time:
https://buymeacoffee.com/cagnulein
https://www.patreon.com/cagnulein/join
https://paypal.me/cagnulein

Software engineer and open source enthusiast, QZ Creator
http://robertoviola.cloud
http://qzfitness.com/


Il giorno dom 8 feb 2026 alle 18:56 simonhaste-svg ***@***.***>
ha scritto:

**Comment by simonhaste-svg** [comment]
I'm trying to connect my bike sx600 to mywhoosh not a rower?

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by cagnulein** [comment]
Hah sorry, simply use another device for mywhoosh and you will see qz as a
bike device there

Let me know


Roberto Viola

I sometimes trade time with my daughter to help. If this was valuable,
please support my time:
https://buymeacoffee.com/cagnulein
https://www.patreon.com/cagnulein/join
https://paypal.me/cagnulein

Software engineer and open source enthusiast, QZ Creator
http://robertoviola.cloud
http://qzfitness.com/


Il giorno dom 8 feb 2026 alle 22:22 simonhaste-svg ***@***.***>
ha scritto:

**Comment by simonhaste-svg** [comment]
So I am connected to qz on phone and mywhoosh on tablet but still can't find my bike ?

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by cagnulein** [comment]
You don't have to see the bike on mywhoosh but the phone where qz runs. Do
you see it?

Roberto Viola

I sometimes trade time with my daughter to help. If this was valuable,
please support my time:
https://buymeacoffee.com/cagnulein
https://www.patreon.com/cagnulein/join
https://paypal.me/cagnulein

Software engineer and open source enthusiast, QZ Creator
http://robertoviola.cloud
http://qzfitness.com/


Il giorno dom 8 feb 2026 alle 23:07 simonhaste-svg ***@***.***>
ha scritto:

**Comment by simonhaste-svg** [comment]
Yes I see all data working on phone but noth9ng happens on my whoosh on tablet when I start pedalling,  it just says cannot find my bike

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by simonhaste-svg** [comment]
Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by cagnulein** [comment]
If you sent images please send on github otherwise I can't see them

**Comment by simonhaste-svg** [comment]
Yes I see all data working on phone but noth9ng happens on my whoosh on tablet when I start pedalling,  it just says cannot find my bike

![Image](https://github.com/user-attachments/assets/f9c4bfe7-c430-4d33-8e97-260c05e61337)

**Comment by cagnulein** [comment]
could you please share an updated debug log so i can check if the settings are right? virtual bluetooth in the experimental settings must be enabled

**Comment by simonhaste-svg** [comment]
![Image](https://github.com/user-attachments/assets/9cb9cd1d-1e61-451f-8ac6-95adbcd48b5b)

![Image](https://github.com/user-attachments/assets/bbe1d38e-c0e8-4275-b8ef-f9f2d05c16f3)

![Image](https://github.com/user-attachments/assets/2036afd0-a9ed-47c8-ada8-c8b4fad35a7c)

**Comment by cagnulein** [comment]
a lot of issue here: fake device must be disabled and also virtual ifit.
Let me know

**Comment by simonhaste-svg** [comment]
OK I'm at work currently so will check when home, hopefully all good and thank you for your help with this.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by cagnulein** [comment]
Ok let me know!

Roberto Viola

I sometimes trade time with my daughter to help. If this was valuable,
please support my time:
https://buymeacoffee.com/cagnulein
https://www.patreon.com/cagnulein/join
https://paypal.me/cagnulein

Software engineer and open source enthusiast, QZ Creator
http://robertoviola.cloud
http://qzfitness.com/


Il giorno lun 9 feb 2026 alle ore 10:17 simonhaste-svg <
***@***.***> ha scritto:

**Comment by simonhaste-svg** [comment]
![Image](https://github.com/user-attachments/assets/6ce38096-19c1-4c39-8bc6-8d63765d76a5)

![Image](https://github.com/user-attachments/assets/1781a52c-dda3-4eab-b264-c1a63a387873)

![Image](https://github.com/user-attachments/assets/1ebf65f6-31df-4566-9097-a17c55f337ed)

Keeps connecting to my phone and going off again

![Image](https://github.com/user-attachments/assets/38f36f46-6972-4ae2-aaef-6bc295356c24)

![Image](https://github.com/user-attachments/assets/f9ef493a-55a8-416b-8ff4-5eee53e4ff60)

**Comment by cagnulein** [comment]
Rename your android device to bike only 4 letters and reboot it 
It usually helps 
Let me know

**Comment by simonhaste-svg** [comment]
Changed it to bike but still same and data has stopped on phone too

![Image](https://github.com/user-attachments/assets/a1c43ba0-710c-4493-a04b-7691b40eb3be)

![Image](https://github.com/user-attachments/assets/15e8da3a-5913-44f1-8f8a-d9611b289f93)

![Image](https://github.com/user-attachments/assets/06090926-6982-43ef-b76e-6129ffec68cf)

**Comment by cagnulein** [comment]
Ok but now the problem is on the phone. Did you try to restart the bike?

Share a fresh debug log if it doesn't work
